src/main/java/baritone/pathing/movement/Movement.java

```diff
@@ -160,11 +163,11 @@ protected boolean prepared(MovementState state) {
                 //i'm doing it anyway
                 //i dont care if theres snow in the way!!!!!!!
                 //you dont own me!!!!
-                state.setTarget(new MovementState.MovementTarget(RotationUtils.calcRotationFromVec3d(player().getPositionEyes(1.0F),
+                state.setTarget(new MovementState.MovementTarget(RotationUtils.calcRotationFromVec3d(ctx.player().getPositionEyes(1.0F),
                         VecUtils.getBlockPosCenter(blockPos)), true)
                 );
                 // don't check selectedblock on this one, this is a fallback when we can't see any face directly, it's intended to be breaking the "incorrect" block
-                state.setInput(InputOverrideHandler.Input.CLICK_LEFT, true);
```

Comment:
I do like these, these are epic

Reply:
Wait what?

Reply:
I like the removal of InputOverrideHandler., thumbs up